packages/lib-classifier/src/components/Classifier/ClassifierContainer.js

```diff
@@ -71,13 +71,18 @@ export default function ClassifierContainer({
 
   const loaded = useHydratedStore(classifierStore, cachePanoptesData, `fem-classifier-${project.id}`)
 
+  useEffect(function onMount() {
+    console.log('resetting stale user data')
+    classifierStore?.userProjectPreferences.reset()
+  }, [])
+
   useEffect(function onLoad() {
-    const { classifications, subjects, workflows } = classifierStore
```

Comment:
This was slightly unsafe, because it will crash if `classifierStore` is undefined. I removed it just to be on the safe side.